robottelo/utils/virtwho.py

```diff
@@ -466,6 +466,32 @@ def hypervisor_json_create(hypervisors, guests):
     return {"hypervisors": hypervisors_list}
 
 
+def hypervisor_fake_json_create(hypervisors, guests):
+    """
+    Create a hypervisor guest json data for fake config usages. For example:
+    {'hypervisors': [{'uuid': '820b5143-3885-4dba-9358-4ce8c30d934e',
+    'guests': [{'guestId': 'afb91b1f-8438-46f5-bc67-d7ab328ef782', 'state': 1,
+    'attributes': {'active': 1, 'virtWhoType': 'esx'}}]}]}
+    :param hypervisors: how many hypervisors will be created
+    :param guests: how many guests will be created
+    """
+    hypervisors_list = []
+    for _ in range(hypervisors):
```

Comment:
Instead of these loops, you can use nested list comprehensions. I think it looks prettier and saves quite a few lines. Non-blocking comment.

```
>>> array_of_arrays = [[{'uuid': uuid.uuid4(), 'something': 'whatever'} for _ in range(3)] for _ in range(4)]

>>> pprint(array_of_arrays)
[[{'something': 'whatever',
   'uuid': UUID('57f5ad3c-91c7-4133-b7ec-f7c2659958bf')},
  {'something': 'whatever',
   'uuid': UUID('e2a9a1c9-8991-420c-b44f-9e5adf0febc7')},
  {'something': 'whatever',
   'uuid': UUID('2d1ced52-f883-4740-bc36-ea13c04037a3')}],
 [{'something': 'whatever',
   'uuid': UUID('3a2710a5-e462-4c2b-90e6-a1867406f9ee')},
  {'something': 'whatever',
   'uuid': UUID('af26c7cc-a873-4b93-adb3-cfc425d018f1')},
  {'something': 'whatever',
   'uuid': UUID('d334ef9c-8152-4fdc-92f7-b3781e8c573d')}],
 [{'something': 'whatever',
   'uuid': UUID('30bd95b5-eab3-4f44-918d-7b072b5f76be')},
  {'something': 'whatever',
   'uuid': UUID('97927b4c-f168-4f19-beb1-ed73712b29b7')},
  {'something': 'whatever',
   'uuid': UUID('b427f66c-9c55-471d-bda0-92abd3e6cdc4')}],
 [{'something': 'whatever',
   'uuid': UUID('98061222-7db5-4b10-9f9a-6c65fab79194')},
  {'something': 'whatever',
   'uuid': UUID('8aa8a817-d3d9-40da-9804-1626d4796520')},
  {'something': 'whatever',
   'uuid': UUID('2dd5b538-f5f0-4f83-80d3-d410c160a235')}]]
>>>
```

Reply:
@lhellebr Have already submit the update ,thanks for your suggestion